02_calculator/calculator.rb

```diff
@@ -1 +1,48 @@
-#write your code here
+# add method takes two parameters and returns the sum
+def add(first_number, second_number)
+  return first_number + second_number
+end   
+
+# subtract method takes in two parameters and returns the answer
+def subtract(first_number, second_number)
+  return first_number - second_number
+end    
+
+# sum method computes the sum of all the numbers sent into this method
+def sum(number_array)
+  total = 0
+  number_array.each do |number|
+    total = total + number
+  end  
+  return total
+end      
+
+# this method multiplies 2 numbers and returns the product
+def multiply(first_number, second_number)
+  return first_number * second_number
+end
+
+# this method takes in array of numbers and multiplies all together and returns the product
+def multiply(*numbers)
```

Comment:
This method should be an array and I think your method only works when prompted like this: **multiply(3,2)** but when you run **multiply([2,3])**, it would not work, therefor I believe you'd have to remove the "*" in the parameter so that it is considered an Array instead of Integers.